test/testdata

```diff
@@ -236,7 +236,8 @@ TD        Gaussian    GaussianTDDFTTest     basicGaussian09       dvb_td.out
 TD        Jaguar      JaguarTDDFTTest       basicJaguar8.3        dvb_td.out
 TD        ORCA        OrcaTDDFTTest         basicORCA2.9          dvb_td.out
 TD        ORCA        OrcaTDDFTTest         basicORCA3.0          dvb_td.out
-TD        QChem       OrcaTDDFTTest         basicQChem4.2         dvb_td.out
+TD        ORCA        OrcaROCISTest         basicORCA4.0          dvb_rocis.out
```

Comment:
It's not so much timing as whether or not the test type is general enough, though we are clearly starting to move away from that, and correctness is the most important thing.